frontend/src/pages/add-blog.tsx

```diff
@@ -15,6 +16,13 @@ type FormData = {
   isFeaturedPost: boolean;
 };
 function AddBlog() {
+  const [selectedImage, setSelectedImage] = useState<string | null>("");
+
```

Comment:
Why is null required here ? "" would be by default, what is the case where the selected image is null ?

Reply:
Fixed